ArcGISRuntimeSDKQt_CppSamples/Layers/BrowseWfsLayers/BrowseWfsLayers.cpp

```diff
@@ -0,0 +1,175 @@
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+#include "BrowseWfsLayers.h"
+
+#include "Map.h"
+#include "MapQuickView.h"
+#include "WfsFeatureTable.h"
+#include "FeatureLayer.h"
+#include "QueryParameters.h"
+#include "SimpleRenderer.h"
+#include "SimpleLineSymbol.h"
+#include "SimpleMarkerSymbol.h"
+#include "WfsService.h"
+#include "WfsLayerInfo.h"
+
+using namespace Esri::ArcGISRuntime;
+
+BrowseWfsLayers::BrowseWfsLayers(QObject* parent /* = nullptr */):
+  QObject(parent),
+  m_map(new Map(Basemap::topographic(this), this))
+{
+  // create WFS Service
+  m_wfsService = new WfsService(QUrl("https://dservices2.arcgis.com/ZQgQTuoyBrtmoGdP/arcgis/services/Seattle_Downtown_Features/WFSServer?service=wfs&request=getcapabilities"),this);
+
+  // once WFS Service is loaded, populate QStringList with layer names for ComboBox
+  connect(m_wfsService, &WfsService::doneLoading, this, [this](Error e)
+  {
+    if (!e.isEmpty())
+      return;
+
+      m_wfsLayersInfoList = m_wfsService->serviceInfo().layerInfos();
+      for(auto i : m_wfsLayersInfoList)
+        m_layerInfoTitleList.append(i.title());
+
+      emit layerInfoTitleListChanged();
+  });
+
+  m_wfsService->load();
+
+  // set initial viewpoint
+//  Envelope env(-122.341581, 47.613758, -122.332662, 47.617207, SpatialReference::wgs84());
+//  m_map->setInitialViewpoint(Viewpoint(env));
+
+}
+
+BrowseWfsLayers::~BrowseWfsLayers() = default;
+
+void BrowseWfsLayers::init()
+{
+  // Register the map view for QML
+  qmlRegisterType<MapQuickView>("Esri.Samples", 1, 0, "MapView");
+  qmlRegisterType<BrowseWfsLayers>("Esri.Samples", 1, 0, "BrowseWfsLayersSample");
+}
+
+MapQuickView* BrowseWfsLayers::mapView() const
+{
+  return m_mapView;
+}
+
+// Set the view (created in QML)
+void BrowseWfsLayers::setMapView(MapQuickView* mapView)
+{
+  if (!mapView || mapView == m_mapView)
+    return;
+
+  m_mapView = mapView;
+  m_mapView->setMap(m_map);
+
+  connect(m_mapView, &MapQuickView::navigatingChanged, this, [this]()
+  {
+    if (m_mapView->isNavigating())
+      return;
+
+    populateWfsFeatureTable();
+  });
+
+  emit mapViewChanged();
+}
+
+void BrowseWfsLayers::createWfsFeatureTable(int index, bool swap)
+{
+  // clear previous layer
+  m_map->operationalLayers()->clear();
+  // set viewpoint to extent of selected layer
+  m_mapView->setViewpointGeometry(m_wfsLayersInfoList[index].extent());
+  // create WFS Feature Table from selected layer
+  m_wfsFeatureTable = new WfsFeatureTable(m_wfsLayersInfoList[index], this);
+
+  // Set feature request mode to manual - only manual is supported at v100.5.
```

Comment:
Should a comment be left, saying what is supported at v100.6?

Reply:
Good question. This is a 100.5 sample so that may be a step too far but I won't take the final say on this. @ldanzinger any thoughts on this?

Reply:
Yes, that's fair enough. 
May I ask, however, about the version number in BrowseWfsLayers.pro which says `ARCGIS_RUNTIME_VERSION = 100.6`: https://github.com/Esri/arcgis-runtime-samples-qt/pull/893/files#diff-03dda30ae078e83b2627972fb6c63cebR22 should it not say `100.5`?

Reply:
This will likely be safe enough - I don't think it will be going into master until 100.6.0.

Reply:
Is there a ticket prepared for updating this comment when we support automatic?